rubygem-concurrent-ruby-edge/rubygem-concurrent-ruby-edge.spec

```diff
@@ -0,0 +1,70 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name concurrent-ruby-edge
+%global rubyabi 1.9.1
+
+Summary: Edge concepts for the modern concurrency tools for Ruby
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.0.pre3
+Release: 1%{?dist}
+Group: Development/Languages
+
+License: MIT
+URL: https://github.com/ruby-concurrency/concurrent-ruby
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: %{?scl_prefix}ruby(abi) = %{rubyabi}
+Requires: %{?scl_prefix}ruby(rubygems)
+Requires: %{?scl_prefix}ruby
+BuildRequires: %{?scl_prefix}ruby(abi) = %{rubyabi}
```

Comment:
Thanks, updated. I've used https://github.com/theforeman/foreman-packaging/blob/rpm/develop/rubygem-pg/rubygem-pg.spec#L21 as a base for the beginning, which we probably don't build for Fedora, so it was not a lucky choice for me.